tests/vmi_lifecycle_test.go

```diff
@@ -49,6 +49,22 @@ func newCirrosVMI() *v1.VirtualMachineInstance {
 	return tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.RegistryDiskFor(tests.RegistryDiskCirros), "#!/bin/bash\necho 'hello'\n")
 }
 
+func addNodeAffinityToVMI(vmi *v1.VirtualMachineInstance, nodes *k8sv1.NodeList) {
```

Comment:
hm why do you send `NodeList` when it the func you are using only `nodes.Items[0].Name`
- or send only one node
- or change the method to add affinity for all nodes under `NodeList`